apps/ui/src/views/Space/Pro.vue

```diff
@@ -228,22 +227,11 @@ const features = computed<Feature[]>(() => {
                 :key="type"
                 class="w-[120px] text-center"
               >
-                <component
-                  :is="feature[type]"
-                  v-if="typeof feature[type] === 'object'"
-                  class="mx-auto text-skin-text"
-                />
-                <ICCheck
-                  v-else-if="feature[type] === true"
-                  class="mx-auto text-skin-success"
-                />
-                <template v-else>
-                  {{
-                    typeof feature[type] === 'number'
-                      ? _n(feature[type])
-                      : feature[type]
-                  }}
-                </template>
+                {{
+                  typeof feature[type] === 'number'
+                    ? _n(feature[type])
+                    : feature[type]
+                }}
```

Comment:
It will always be a number here
```suggestion
                {{ _n(feature[type]) }}
```